hw1/02-palindrome.js

```diff
@@ -1,3 +1,36 @@
-const elem = document.querySelector('input');
+const section = document.querySelector("section");
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+//Input event listener
+elem.addEventListener("input", handleInput);
+
+//Create a new div for the output
+const outputDiv = document.createElement("div");
+outputDiv.setAttribute("id", "output");
+section.append(outputDiv);
+
+//Handle the event listener
+function handleInput(event) {
+  event.preventDefault();
+  const input = event.target.value;
+  let reverseInput = input.split("").reverse().join("");
+  console.log(input);
+
+  if (input < 0) {
+    outputDiv.style.color = "red";
```

Comment:
As part of the Airbnb style guide, you should use single quotes rather than double quotes. Double quotes are used throughout this document, but I'm just going to mention it here.